### ISSUE TITLE
Why is the scrollbar being reset globally?

### ISSUE THREAD

**Issue by mthezi** [aidenybai/react-scan#42]
This library will globally change the appearance of scrollbars!
```css
* {
    scrollbar-width: thin;
    scrollbar-color: rgba(255, 255, 255, 0.3) rgba(255, 255, 255, 0.1);
}
```

I spent an hour troubleshooting the scrollbar issue on my page!

**Comment by alexgarabt** [comment]
> Duplicate of [#42](https://github.com/aidenybai/react-scan/issues/42), I also been some time checking the styles on my project to see what it fails. I will try to make a pull request to fix the bug

I have made a [pull request](https://github.com/aidenybai/react-scan/pull/44) to fix the bug.

**Comment by aidenybai** [comment]
Fixed in 0.0.32 @alexgarabt @mthezi
